CONTRIBUTING.md

```diff
@@ -0,0 +1,95 @@
+# Contributing to Arbor
+
+First off, thank you for considering contributing to Arbor! It's people like you that
+make Arbor such a great tool. Feel welcome and read the following sections in order to
+know how to ask questions and how to work on something.
+
+## Types of contributions
+
+There are many ways to contribute, from writing tutorials or blog posts, improving the
```

Comment:
Minor suggestion: maybe replace `ways to contribute, from writing tutorials or blog posts` with `ways to contribute: writing tutorials or blog posts`